data/pathfinder/paizo/player_companion/magical_marketplace/mm_equipmods.lst

```diff
@@ -9,21 +9,21 @@ SOURCELONG:Pathfinder Player Companion: Magical Marketplace	SOURCESHORT:MM	SOURC
+Mind Buttressing											KEY:Special Ability ~ Mind Buttressing ~ Armor	TYPE:Armor					PLUS:2			VISIBLE:QUALIFY	SOURCEPAGE:p.7											PRETYPE:2,ArmorEnhancement,Armor			!PRETYPE:2,Armor,Light		SPROP:+2 resistance bonus on Will saves and renders her immune to possession and mental control.
+Poisoning												KEY:Special Ability ~ Poisoning ~ Armor		TYPE:Armor					PLUS:1			VISIBLE:QUALIFY	SOURCEPAGE:p.27											PRETYPE:2,ArmorEnhancement,Armor								SPROP:3 rounds after adding a vial of poison, the armor exposes any creature grappling (if injury poison) or biting (if ingested poison) the wearer to said poison.
+Rebounding												KEY:Special Ability ~ Rebounding ~ Shield		TYPE:Shield					PLUS:1			VISIBLE:QUALIFY	SOURCEPAGE:p.21											PRETYPE:2,ShieldEnhancement,Shield								SPROP:Provides an additional +2 bonus to AC against thrown weapons, and it adds its total bonus to touch AC against splash weapons. If targeted by a splash weapon that misses, the wielder may choose the misdirection of the splash weapon.
 
 # ==============================
 # New Weapon Special Abilityies
 # ==============================
 
-Legbreaker												KEY:Special Ability ~ Legbreaker ~ Melee		TYPE:Weapon.Melee.Bludgeoning		PLUS:2			VISIBLE:QUALIFY	SOURCEPAGE:p.15											PRETYPE:3,Enhancement,Weapon,Bludgeoning						SPROP:Reduces the base land speed of a struck creature by 10 feet for 1d4 rounds.
-Penetrating												KEY:Special Ability ~ Penetrating ~ Ammunition	TYPE:Ammunition				PLUS:2			VISIBLE:QUALIFY	SOURCEPAGE:p.23											PRETYPE:2,Enhancement,Ammunition							SPROP:If target fails DC 17 Fort save, ammunition travels through target. May hit second, in-line target at -4 to hit.
-Rusting												KEY:Special Ability ~ Rusting ~ Weapon		TYPE:Ammunition.Weapon.Melee.Ranged	PLUS:1			VISIBLE:QUALIFY	SOURCEPAGE:p.15											PRETYPE:1,Enhancement									SPROP:On a critical hit, you may forgo any extra damage to instead deal normal damage to the target and affect the target or their armor with the rusting grasp spell.
+Legbreaker												KEY:Special Ability ~ Legbreaker ~ Melee		TYPE:Weapon.Melee.Bludgeoning		PLUS:2			VISIBLE:QUALIFY	SOURCEPAGE:p.15											PRETYPE:3,WeaponEnhancement,Weapon,Bludgeoning						SPROP:Reduces the base land speed of a struck creature by 10 feet for 1d4 rounds.
+Penetrating												KEY:Special Ability ~ Penetrating ~ Ammunition	TYPE:Ammunition				PLUS:2			VISIBLE:QUALIFY	SOURCEPAGE:p.23											PRETYPE:2,WeaponEnhancement,Ammunition							SPROP:If target fails DC 17 Fort save, ammunition travels through target. May hit second, in-line target at -4 to hit.
+Rusting												KEY:Special Ability ~ Rusting ~ Weapon		TYPE:Weapon.Melee.Ranged	PLUS:1			VISIBLE:QUALIFY	SOURCEPAGE:p.15											PRETYPE:2,WeaponEnhancement,Weapon									SPROP:On a critical hit, you may forgo any extra damage to instead deal normal damage to the target and affect the target or their armor with the rusting grasp spell.
```

Comment:
Ammunition TYPE removed for some reason.

Reply:
Rusting is not actually allowed to be an ammunition.
![Screenshot_2019-11-19_23-14-06](https://user-images.githubusercontent.com/130179/69216980-57ea7c00-0b22-11ea-837a-3c76e31c4dc3.png)